### ISSUE TITLE
Introduction -> mention the evolution of the training courses followed by new recruits ?

### ISSUE THREAD

**Issue by mpjoubertdebellefon** [ThomasFaria/retex-innovation-insee#27]
When you write 

> Not incidentally, an increasing number of public statisticians trained as data scientists have joined NSOs in recent years

I suggest to mention the fact that this arrival of data scientists is due to the natural evolution of their training -> it is not that NSOs changed their recrutment channel.  Something like that "Due to the evolution of training courses followed by new recruits"...

**Comment by avouacr** [comment]
Not sure to agree there. It seems to me that it is partly due to changes in initial training, and also partly due to changes in the profiles being recruted. Surely people are more frequently trained to data science methods in statistics schools, but we could have chosen to keep recruiting economists or survey specialists for instance, instead of people identifying themselves as datascientists

**Comment by RLesur** [comment]
This is a very complex debate. It seems to me that most curricula that include quantitative training now have courses in machine learning (in addition to traditional econometric courses). This is a natural evolution driven by the market and the evolutions in the data field.  

But most NSIs have recognized the importance of building capabilities in data science (we still struggle to implement data science methods correctly, but that is another topic). They are training their employees in data science and hiring some data scientists and data engineers in addition to economists, demographs or survey specialists (who are increasingly familiar with data science without actually being data scientists).

To summarize, my opinion is that both trends are true. I wouldn't say that data science adoption is only driven by new recruits. Most NSIs want to embrace data science, to varying degrees.  
We can see this evolution in the curricula of the European Masters in Official Statistics (EMOS), which are not, strictly speaking, masters in data science. EMOS students have courses in machine learning, following the needs expressed by ESS bodies.
